pkgs/development/libraries/libvorbis/default.nix

```diff
@@ -12,6 +12,7 @@ stdenv.mkDerivation rec {
 
   patches = [
     (fetchpatch {
+      # Also fixes CVE-2018-10393.
```

Comment:
It's better to all CVEs in the patch name so it's detected by tools like `vulnix` and https://broken.sh.

Reply:
At the very least Vulnix only check the instantiated derivation, not the expression.

Reply:
Done, switched to the canonical "CVE...+CVE..." filename format.

Reply:
After this we need to switch the PR to `staging` though because the name change is a mass rebuild.

Reply:
> After this we need to switch the PR to `staging` though because the name change is a mass rebuild.

I've done this now.